coalib/bearlib/abstractions/Lint.py

```diff
@@ -1,339 +0,0 @@
-import os
-import re
-import shutil
-from subprocess import check_call, CalledProcessError, DEVNULL
-import tempfile
-
-from coalib.bears.Bear import Bear
-from coalib.misc.Decorators import enforce_signature
-from coalib.misc.Shell import escape_path_argument, run_shell_command
-from coalib.results.Diff import Diff
-from coalib.results.Result import Result
-from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
-
-
-class Lint(Bear):
```

Comment:
how about keeping legacy lint as a deprecated class and emit a warning on the debug channel upon execution? Then we don't break all bears right?